docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_intro.md

```diff
@@ -0,0 +1,29 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_intro.md"]
+```
+
+# Introduction
+
+Let $\mathbb{F}$ be an ordered field; the most prominent case here is where $\mathbb{F}=\mathbb{Q}$ are the rational numbers.
```

Comment:
"the most prominent case" -- wha it that supposed to mean? Prominent with whom and for what, and why is it relevant for the reader?

Perhaps you wanted something like this?
```suggestion
Let $\mathbb{F}$ be an ordered field, for example the field $\mathbb{Q}$ of rational numbers.
```

Reply:
Also, this text seems to replicate things that were started in PR #508 -- how do that PR and the one here relate to each other?

Reply:
Within a common zoom session we merged that branch into this docs branch. I will close #508.